sdk/README.md

```diff
@@ -18,33 +17,67 @@ a basic sequential pipeline.
 
 ## Steps
 
-1. Setup Python environment with Conda or a Python virtual environment:
+1. Clone the kfp-tekton repo:
+    - `git clone https://github.com/kubeflow/kfp-tekton.git`
+    - `cd kfp-tekton`
+
+
+2. Setup Python environment with Conda or a Python virtual environment:
 
     - `python3 -m venv .venv`
     - `source .venv/bin/activate`
 
-2. Build the compiler:
+3. Build the compiler:
 
     - `pip install -e sdk/python`
 
-3. Run the compiler tests (optional):
+4. Run the compiler tests (optional):
 
     - `./sdk/python/tests/run_tests.sh`
 
-4. Compile the sample pipeline:
+5. Compile the sample pipeline:
 
     - `cd sdk/samples`  
     - `dsl-compile-tekton --py parallel_join.py --output pipeline.yaml`
     
-5. Run the sample pipeline on a Tekton cluster:
+6. Run the sample pipeline on a Tekton cluster:
 
     - `kubectl apply -f pipeline.yaml`
     - `tkn pipeline start parallel-pipeline`
```

Comment:
we can skip step 7 if we change
-  `tkn pipeline start parallel-pipeline`
to
- `tkn pipeline start parallel-pipeline  --showlog`

which will show logs right after starting the pipeline. 

That is the behavior I saw in `tkn` `0.6.0` when I created this instruction (`--showlog` was implicit)
`